tools/sgcollect_info

```diff
@@ -199,7 +199,10 @@ def extract_element_from_config(element, config):
     sync_regex = r'"Sync":(`.*`)'
     config = re.sub(sync_regex, '"Sync":""', config)
     try:
-        return json.loads(config)[element]
+        # convert dictionary keys to lower case
+        lower_case_keys_dict = {k.lower():v for k,v in json.loads(config).items()}
```

Comment:
What does `k.lower():v` do?  Can this be changed to be less cryptic?  (maybe pull out to a named variable, even if it means unrolling this dictionary comprehension)